app/grandchallenge/workstation_configs/models.py

```diff
@@ -291,6 +292,12 @@ class ImageInterpolationType(models.TextChoices):
         help_text="Default color for displaying and creating annotations",
     )
 
+    default_annotation_line_width = PositiveSmallIntegerField(
+        blank=True,
+        null=True,
+        help_text="Default line width for displaying and creating annotations",
```

Comment:
Are there any units to this? mm, voxels, px, cm, mm, in, pt, pc etc etc?

Reply:
It is not obvious from any documentation, but some practical interactions would suggest it is pixels. I've updated the help text.